StortSpelProjekt/Engine/src/Renderer/Transform.cpp

```diff
@@ -95,3 +95,13 @@ float3 Transform::GetPositionFloat3() const
 	pos.z = m_Position.z;
 	return pos;
 }
+
+DirectX::XMFLOAT3 Transform::GetScale() const
+{
+	return m_Scale;
+}
+
+DirectX::XMMATRIX Transform::GetRotMatrix() const
+{
+	return m_RotationMat * m_RotXMat * m_RotYMat * m_RotZMat;
+}
```

Comment:
This function should only return the rotationMatrix and not do the math. Since this function should possibly be able to be called multiple times each frame without having to re-do the math calculations.

Reply:
fixed